docs/conf.py

```diff
@@ -71,19 +71,16 @@
 copyright = nengo_spa.__copyright__
 version = '.'.join(nengo_spa.__version__.split('.')[:2])  # Short X.Y version
 release = nengo_spa.__version__  # Full version, with tags
-pygments_style = 'default'
+pygments_style = 'friendly'
 
 # -- Options for HTML output --------------------------------------------------
 
-html_theme = 'sphinx_rtd_theme'
-html_theme_path = [sphinx_rtd_theme.get_html_theme_path()]
+html_theme = 'nengo_sphinx_theme'
 html_title = "Nengo SPA {0} docs".format(release)
 html_static_path = ['_static']
 html_favicon = os.path.join('_static', 'favicon.ico')
-html_logo = os.path.join('_static', 'square-light.svg')
-html_context = {
-    'css_files': [os.path.join('_static', 'custom.css')],
-}
+html_logo = os.path.join('_static', 'full-light.svg')
```

Comment:
The logo is handled differently in Nengo Sphinx Theme so that we don't have to include the `svg` files in every repository; set the `nengo_logo` key in `html_theme_options` to `nengo-spa-full-light.svg` (see https://github.com/nengo/nengo/blob/master/docs/conf.py#L89 as an example).

You should also set `nengo_logo_color` to #d40000 and the set `analytics_id` to be the same ID as the Nengo core one so that we can track user activity across all of the documentation page.

Reply:
done